systemtests/scripts/functions

```diff
@@ -1110,6 +1110,16 @@ if [ "$USER" == "root"  ]; then
 fi
 }
 
+# Used by tlsrestricted
+skip_if_openssl_too_old()
+{
+openssl_version=$(openssl version | cut -d " " -f2 | sed 's/\([0-9]\)\.\([0-9]\).*/\1\2/' || 0)
+if [ ${openssl_version} -le 10  ]; then
+ echo "${TestName} test skipped: test cannot be run without openssl > 1.1x"
+ exit 77;
+fi
+}
+
```

Comment:
You can just check the OPENSSL_VERSION in CMake and disable the test. Should be a lot easier.
```
if(OPENSSL_VERSION VERSION_LESS "1.1.0")
  somehow_set_test_disabled()
endif()
```